python/triton/experimental/gluon/language/amd/_layouts.py

```diff
@@ -49,6 +49,7 @@ def __post_init__(self):
         super().__setattr__("cta_split_num", _unwrap_if_constexpr(self.cta_split_num))
         super().__setattr__("cta_order", _unwrap_if_constexpr(self.cta_order))
 
+        assert self.elem_type is ttgl.float32 or self.elem_type is ttgl.float64, "The element type in AMDMFMALayout should be flot or double type"
```

Comment:
typo: flot. I think using `float32 or float64 type` here might be better, given that in triton they are named that way.

Reply:
Also `assert self.elem_type in (ttgl.float32, ttgl.float64)` might be simplier